hack/testUnitKustomizePlugins.sh

```diff
@@ -52,9 +52,7 @@ function scanDir {
 
 if onLinuxAndNotOnRemoteCI; then
   # Some of these tests have special deps.
-  make $MYGOBIN/helmV2
   make $MYGOBIN/helmV3
-  make $MYGOBIN/helm
```

Comment:
neither old version seems to be used by anything anymore, so I removed the targets for these